### ISSUE TITLE
A11y_VisualStudioCodeClient_Editor_ Screen reader: Voiceover is announcing same information for both the suggestions for exclamation mark(!)

### ISSUE THREAD

**Issue by kjawale99** [microsoft/vscode#164700]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; #Voiceover; #Screenreader.

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor 
2. Create a HTML file.
3. Type (!) exclamation mark .
4. Suggestions will appear.
5. Turn on Voiceover
6. Verify that Voiceover is announcing same information for both the suggestions for exclamation mark(!).

### Actual Result:
Voiceover is announcing same information for both the suggestions for exclamation mark(!).

### Expected Result:
Voiceover should announce information related to the suggestion. Voiceover should not announce the same information for both the suggestions.

### Attachment:
![exclamation](https://user-images.githubusercontent.com/98745784/198011683-5ba9bf72-3fd3-4932-a4a5-e74544fc3071.gif)

![revscode (1)](https://user-images.githubusercontent.com/98745784/198995885-c7df4b59-d8cc-4acf-bfd5-23849e616b7f.gif)

**Comment by isidorn** [comment]
This gif is super low quality and I do not see what is going, can you please upload a gif that is clearly showing the problem please? Thanks

**Comment by kjawale99** [comment]
@isidorn I have attached the gif of a better quality than before .

**Comment by isidorn** [comment]
Ok, I see the gif thanks. And what is the issue? VoiceOver reads the label of the item which is `!` or `!!!` and then reads the right hand side which is `Emmet Abbreviation`. What do you expect VoiceOver to read and why is that better for the user?

**Comment by kjawale99** [comment]
@isidorn In speech viewer the ! and !!! are visible but its not announcing about the exclamation.

**Comment by isidorn** [comment]
@kjawale99 well that is a VoiceOver configuration. VS Code can not control this. I suggest you file a new issue for VoiceOver https://bugs.webkit.org/
